infra/scripts/generate_cipher_and_sign_keys/src/main.rs

```diff
@@ -6,4 +6,8 @@ fn main() {
     let cipher_sk = hex::encode(cipher_sk.to_bytes());
     println!("cipher public key: {}", cipher_pk);
     println!("cipher private key: {}", cipher_sk);
+    let sign_sk = near_crypto::SecretKey::from_random(near_crypto::KeyType::ED25519);
```

Comment:
same here. The latest develop should already have these and the new near account keys.

Reply:
I think after merging develop, you need to delete `infra/scripts/generate_cipher_and_sign_keys` now